examples/profile.php

```diff
@@ -70,89 +69,412 @@
             <?php else: ?>
                 <table class="table table-sm table-bordered table-hover">
                     <thead>
-                    <tr>
-                        <th scope="col">Name</th>
-                        <th scope="col">Value</th>
-                        <th scope="col">Sources</th>
-                        <th scope="col">Verifiers</th>
-                    </tr>
+                        <tr class="table-primary">
+                            <th scope="col">Name</th>
+                            <th scope="col">Value</th>
+                            <th scope="col">Anchors</th>
+                        </tr>
                     </thead>
                     <tbody>
                         <tr>
                             <th scope="row">Given Name(s)</th>
                             <td><?php echo $givenNames ? $givenNames->getValue() : '' ?></td>
                             <td>
```

Comment:
We also have the Yoti ID/`rememberMeID` for .NET, and I think some of the others, but obviously not showing anything for the anchors, just `N/A`

Reply:
I don't think we need to display rememberMeId here.